components/brave_wallet/common/common_utils.cc

```diff
@@ -30,7 +31,8 @@ namespace {
 bool IsDisabledByPolicy(PrefService* prefs) {
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
   DCHECK(prefs);
-  return prefs->IsManagedPreference(prefs::kDisabledByPolicy) &&
+  return (BraveOriginState::GetInstance()->IsBraveOriginUser() ||
+          prefs->IsManagedPreference(prefs::kDisabledByPolicy)) &&
```

Comment:
This is what triggered the @brave/crypto-wallets-core review; basically it's checking:

`Is this disabled by policy?`
**AND**

`If person has group policy in place`
**OR**
`If person is a Brave Origin user`


With Brave Origin, we're re-using the group policy preferences (ex: `prefs::kDisabledByPolicy` ) to hide those parts of the UI 😄 

cc: @supermassive

Reply:
Why is it so complicated for wallet and for other components? :)

Should that be just `return prefs->GetBoolean(prefs::kDisabledByPolicy)`? 
Either it is disabled by managed prefs or by brave-origin page - we don't care and wallet is just disabled.
I'd also rename this function from `IsDisabledByPolicy()` to `IsDisabled()` as caller most likely does not need to know why wallet is disabled. And if we go further we'd better avoid negation and use form `IsWalletEnabled()` to be used everywhere as something like
```
if (!IsWalletEnabled()) {
  return;
}

```

Reply:
Why is it so complicated for wallet and for other components? :)

Should that be just `return prefs->GetBoolean(prefs::kDisabledByPolicy)`? 
Either it is disabled by managed prefs or by brave-origin page - we don't care and wallet is just disabled.
I'd also rename this function from `IsDisabledByPolicy()` to `IsDisabled()` as caller most likely does not need to know why wallet is disabled. And if we go further we'd better avoid negation and use form `IsWalletEnabled()` to be used everywhere as something like
```
if (!IsWalletEnabled()) {
  return;
}

```